news/asciidoctorj-1-6-0-released.adoc

```diff
@@ -0,0 +1,118 @@
+= AsciidoctorJ 1.6.0 released, at last!
+Robert Panzer; Dan Allen
+2018-12-31
+:page-tags: [asciidoctorj, release]
+:url-project: https://asciidoctor.org/docs/asciidoctorj
+:url-asciidoctor: https://asciidoctor.org
+:url-leanpub-converter: https://github.com/asciidoctor/asciidoctor-leanpub-converter
+:url-integrator-guide: https://github.com/asciidoctor/asciidoctorj/blob/master/docs/integrator-guide.adoc
+:url-maven-plugin: https://github.com/asciidoctor/asciidoctor-maven-plugin
+:url-gradle-plugin: https://github.com/asciidoctor/asciidoctor-gradle-plugin
+:url-jpms: http://openjdk.java.net/projects/jigsaw/
+:url-log-handling-api: https://github.com/asciidoctor/asciidoctorj/blob/master/docs/integrator-guide.adoc#logs-handling-api
+:url-converter-api: https://github.com/asciidoctor/asciidoctorj/blob/master/docs/integrator-guide.adoc#publish-everywhere-adapt-asciidoctor-to-your-own-target-format
+
+After several years of brewing (or perhaps barrel aging?), we finally released AsciidoctorJ 1.6.0!
+
+.What is AsciidoctorJ?
+****
+{url-project}[AsciidoctorJ] is the official library for running Asciidoctor on the JVM.
+With AsciidoctorJ, you can convert AsciiDoc content, analyze the structure of a parsed AsciiDoc document, and write Asciidoctor extensions in Java and other JVM languages.
+****
+
+[.lead]
+Before you upgrade, there are some important things you need to know, which this post covers.
+While still built on the strong foundation of {url-asciidoctor}[Asciidoctor] (the RubyGem) underneath, AsciidoctorJ has matured into a project in its own right.
+This release brings stability and key enhancements to new and existing users alike.
+Grab a glass so we can pour you all the details.
+
+== A major stepping-stone to SemVer
+
+*Despite the version number, this release should be considered a major (breaking) release.*
+
+Asciidoctor 1.6.0 is a transitional release in preparation for making the switch to Semantic Versioning (or SemVer, for short) in 2.0.0.
+SemVer is a logical system for choosing version numbers that uses a simple MAJOR.MINOR.PATCH sequence.
+A MAJOR version is for when you make incompatible changes.
```

Comment:
The "you" in lines 35, 36, and 37, makes it feel like, me, the reader will be making these releases :wink:. I'd suggest either changing the "you" to "we" or removing the pronoun all together. For example: A MAJOR version is for introducing incompatible changes.

Reply:
I can definitely get behind @Mogztter 's comment about trimming down the SemVer section. You could just lead with the sentence on line 33 and then link them to the SemVer blog post for more info.